index.js

```diff
@@ -3023,10 +3001,9 @@
 
   //# anyPass :: Array (a -> Boolean) -> a -> Boolean
   //.
-  //. Takes an array of unary predicates and a value of any type
-  //. and returns `true` if any of the predicates pass; `false` otherwise.
-  //. None of the subsequent predicates will be evaluated after the
-  //. first passed predicate.
+  //. Takes an array of predicates and a value of any type and returns `true`
+  //. [iff][] the value satisfies any of the predicates. None of the subsequent
+  //. predicates will be applied after the first predicate satisfied.
```

Comment:
I made use of [iff][1] here then decided to make some other changes to the `S.allPass` and `S.anyPass` descriptions while I was at it.


[1]: https://en.wikipedia.org/wiki/If_and_only_if